cmake/kokkos_enable_options.cmake

```diff
@@ -40,6 +40,7 @@ KOKKOS_ENABLE_OPTION(DEPRECATION_WARNINGS ON "Whether to emit deprecation warnin
 KOKKOS_ENABLE_OPTION(HIP_RELOCATABLE_DEVICE_CODE  OFF "Whether to enable relocatable device code (RDC) for HIP")
 KOKKOS_ENABLE_OPTION(HPX_ASYNC_DISPATCH   OFF "Whether HPX supports asynchronous dispatch")
 KOKKOS_ENABLE_OPTION(TESTS         OFF  "Whether to build the unit tests")
+KOKKOS_ENABLE_OPTION(PERF_TESTS    OFF  "Whether to build the perf tests")
```

Comment:
Why "performance tests" and not "benchmarks"?

Reply:
I agree that we should use the term benchmarks, both in displayed messages and option names (such as Kokkos_ENABLE_BENCHMARKS).

One term of art is better than two, and "benchmarks" is better than "performance tests".

Reply:
Option renamed to `Kokkos_ENABLE_BENCHMARKS`.